options/locale/locale_en-US.ini

```diff
@@ -688,6 +688,7 @@ emails = Email Addresses
 manage_emails = Manage Email Addresses
 manage_themes = Select default theme
 manage_openid = Manage OpenID Addresses
+email_cant_add = You cannot add any new email addresses at this time. Please try again later.
```

Comment:
Could you remove the "try again later"? As that may create confusion for the user, and they'll try to refresh the page without knowing why they can't add a new email. From the ticket you linked it sounds like the user hadn't verified their email yet.